### DISCUSSION TITLE
Next `jupyter-book` release?

### DISCUSSION THREAD

**Discussion by deppen8** [executablebooks/meta#893] [Q&A]
Sorry if this information exists somewhere, but I couldn't find it. I'm wondering when the next release of `jupyter-book` might be. I'm hoping to be able to use the changes to dependencies in [this PR](https://github.com/executablebooks/jupyter-book/pull/1871) to make my life a little easier.

**Comment by choldgraf** [comment]
Hey @deppen8 - there were a *bunch* of changes under the hood (some breaking) in MyST-NB, parser, and the book theme. The plan right now is something like:

- Merge remaining changes in the book theme: https://github.com/executablebooks/sphinx-book-theme/pull/640
- De-buggify and merge the updates of all these packages: https://github.com/executablebooks/jupyter-book/pull/1842
- Cut a new release after a release candidate or two

I wonder if we should instead just cut a quick release now before all of that happens though, to bring in minor updates like the PR you mentioned. @mmcky what do you think?
